.gitignore

```diff
@@ -23,3 +23,5 @@ dist/
 test_output
 
 **/generated-plugin
+
+!.github/ISSUE_TEMPLATES/config.yml
```

Comment:
Should this file be in the current PR also?

Reply:
Yes. Otherwise the config gets ignored.

Reply:
Exclamation point prefix means "don't ignore"

Reply:
Whoops. That needs to be ISSUE_TEMPLATE. SINGULR.